Build.ps1

```diff
@@ -21,12 +21,12 @@ $buildSuffix = @{ $true = "$($suffix)-$($commitHash)"; $false = "$($branch)-$($c
 echo "build: Package version suffix is $suffix"
 echo "build: Build version suffix is $buildSuffix" 
 
-foreach ($src in ls src/*) {
+foreach ($src in dir src/*) {
```

Comment:
Since we're changing this anyway, should we just avoid using the aliases? They give me a lot of headaches.

Reply:
Makes sense